### ISSUE TITLE
Incomplete implement of multiple artist tag in some formats

### ISSUE THREAD

**Issue by mlny** [CDrummond/cantata#916]
Platform: macOS and Windows
Version: 2.0.1

Hi,

If I tag multiple artist in FLAC/Ogg Vorbis, Cantata would show last artist only. Could you support FLAC/Ogg Vorbis multiple artist tag?

I have tested some format, and found multi-artist tagged MP3s are supported by MPD itself, showed as combined. But with FLAC/Vorbis, artist field splitted.(In fact, GMPC support this format and showed combined result.) However, ALAC and AAC doesn't support multi-artist at all, even showed as splitted in foobar2000.

(Detailed test result: http://imgur.com/a/e0nqI )

Thanks for this awesome client. Sorry for bad English, I'm not good at English.

**Comment by CDrummond** [comment]
I can do this easily enough. But how?
Album = X
Artist = A
Artist = B

Should Cantata show:
Artist = A, B
  Album = X

Or:
Artist = A
  Album = X
Artist = B
  Album = X

???

**Comment by mlny** [comment]
Well, I have no problem with grouping thanks to album artist.

I didn't tested search, so I did and found search works without problem - it's just a visual problem.

![snapcrab_cantata_2016-11-13_11-46-27_no-00](https://cloud.githubusercontent.com/assets/5173607/20243345/e9e58578-a996-11e6-800d-0ff47e8e35bf.png)


So it's good to do with
```
Artist = A, B
Album = X
```
format.

And if they not have album artist tagged, group as a single "Various Artists" album is good I think, but it seems more complex than simple fallback(I'm not a programmer, so I can't estimate difficulty to do this).

(If whole album have multi-artist song, its album artist should tagged as "Various Artists" or its lead artist and if not, it's user-side problem. I think it's just a "looks weird" problem.)

**Comment by CDrummond** [comment]
To be honest, not sure I see the point to this. Cantata will **not** group tracks with different artists under 'Various Artists' - this is up to you to set the AlbumArtist tag.

So, if you have album-artist set it will group by that. If you have:

Album X
Track 1 - Artist A
Track 2 - Artist B
...but no album artist tag, then you will get 2 albums - one for each artist. Cantata cannot group on album name only, as you can have multiple 'Greatest Hits' by different artists.

What I was proposing above would **only** work if **all** tracks had the same 2 artists, and no album-artist set. So you would have 'Artist A, Artist B' with 'Album X'

However, if you want this - then why not just set AlbumArtist to 'Artist A, Artist B' ???

**Comment by mlny** [comment]
Okay sorry to making confusion. Now it's clear to what you're saying - it's simple approach and will works well.

> However, if you want this - then why not just set AlbumArtist to 'Artist A, Artist B' ???

I thought some lazy guys don't want to tag album artist, but I also wrote: 
> If whole album have multi-artist song, its album artist should tagged as "Various Artists" or its lead artist and if not, it's user-side problem.

So it's up to me and users. I'm happy with" Various Artists"(As I showed, search works flawlessly even it's not showed as multi-artists. Grouping is also correct.) and it's not a problem.
